overlord/ifacestate/helpers.go

```diff
@@ -393,7 +393,7 @@ ConnsLoop:
 			} else {
 				logger.Noticef("cannot refresh static attributes of the connection %q", connId)
 			}
-		case "system-files":
+		case "system-files", "shared-memory":
```

Comment:
I really agree with this. There is no logic in why we would just sync some, and not all. - Also it's problematic the code is here, in the middle of some logic loop for reloading connections (in the sense we're going to forget this exists again).

Reply:
Yeah! Soooooo true. Please add a backlog jira card that refers to this PR with a small comment that we need to revisit it :)

Reply:
there's a PR from field that addresses this more broadly https://github.com/snapcore/snapd/pull/12878

Reply:
sadly we can't just do this, we need to do enough checks similarly like the content case does some checks. system-files is different because it is always an implicit slot that is blocked by installation so it's less of a concern